experiments/azure/cyclecloud/cpu/experiment/data/amg2023/logs-08-21-2024/amg-2n-err.log

```diff
@@ -0,0 +1,3 @@
+real 141.63
```

Comment:
Are these from cyclecloud or parallel cluster?

Reply:
They should be removed, and are now.